src/api/interface.ts

```diff
@@ -102,34 +103,36 @@ declare class Channel {
    * Note that this function can be used without first joining the channel, allowing applications to broadcast on
    * channels that they aren't a member of.
    * 
-   * This broadcast will be received by all windows that are members of this channel, *except* for the window that
-   * makes the broadcast. This matches the behaviour of the top-level FDC3 `broadcast` function.
```

Comment:
@nkolba I have removed this part - seems like an implementation detail to me. FDC3 has no concept of "windows", so it shouldn't be in there.